nntrainer/layers/multi_head_attention_layer.h

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2022 hyeonseok Lee <hs89.lee@samsung.com>
+ *
+ * @file   multi_head_attention_layer.h
+ * @date   08 July 2022
+ * @see    https://github.com/nnstreamer/nntrainer
+ *         https://arxiv.org/abs/1706.03762
+ * @author hyeonseok Lee <hs89.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is MultiHeadAttention Layer Class for Neural Network
+ *
+ */
+
+#ifndef __MULTI_HEAD_ATTENTION_LAYER_H__
+#define __MULTI_HEAD_ATTENTION_LAYER_H__
+#ifdef __cplusplus
+
+#include <acti_func.h>
+#include <layer_impl.h>
+
+namespace nntrainer {
+
+/**
+ * @class   Multi Head Attention Layer
+ * @brief   Implementation of multi head attention which is described in paper
+ * "Attention is all you need"
```

Comment:
How about adding url for paper? https://arxiv.org/abs/1706.03762

Reply:
Already described at line 8